modules/modify-data-object/pkg/dataobject/provider.go

```diff
@@ -72,6 +72,7 @@ func (d *dataObjectProvider) DeletePVC(namespace string, name string) error {
 }
 
 func waitUntilDataObjectIsDeleted(helper *resource.Helper, namespace, name string) {
+	log.Logger().Info("waiting until Data object is deleted")
```

Comment:
nit: Use same case in all messages? `data object` or `Data object`

Reply:
fixed